BREAKING.md

```diff
@@ -57,6 +60,39 @@ getFileLink(
 )
 ```
 
+### AgentScheduler removed as a default member of ContainerRuntime
+Prior to 0.38, `ContainerRuntime` would automatically add an `AgentScheduler` to the container upon creation.  This is no longer added by default.  If you support back compat with documents created prior to 0.38, they will require the `AgentScheduler` to be present in the registry for compatibility.
+
+To facilitate migration, `makeContainerRuntimeWithAgentScheduler()` is provided in the `@fluidframework/deprecated-agent-scheduler-container-runtime` package that produces a runtime which is compatible with the old `ContainerRuntime.load()` by adding an `AgentScheduler`.  This helper is deprecated for new use and will be removed in an upcoming release.  Container authors should plan to incorporate the `AgentScheduler` into their container code if needed, and move to the new `ContainerRuntime.load()`.
+
+`AgentSchedulerBaseContainerRuntimeFactory` and `AgentSchedulerContainerRuntimeFactoryWithDefaultDataStore` are also provided for back-compat with the aqueduct container runtime factories.  Similarly to `makeContainerRuntimeWithAgentScheduler()`, container authors should plan to incorporate the `AgentScheduler` into their subclasses (adding it to the registry and instantiating on `containerInitializingFirstTime()`).
+
+### Leader API surface removed
+Since the `AgentScheduler` is no longer built into `ContainerRuntime`, the leadership election functionality which depended upon it is also no longer built into the runtime.  For similar functionality, you can use the `TaskSubscription` from the `@fluidframework/agent-scheduler` package which exposes a `.haveTask()` similar to the `.leader` property and `"gotTask"`/`"lostTask"` similar to the `"leader"`/`"notleader"` events previously available on the `ContainerRuntime` and data store interfaces.
+
+### IContainerRuntimeDirtyable removed
+Since the `AgentScheduler` is no longer built into `ContainerRuntime`, the `ContainerRuntime` no longer excludes any messages from dirty consideration.  As a result, `IContainerRuntimeDirtyable` has been removed and `ContainerRuntime` no longer implements it.  If you currently check `IContainerRuntimeDirtyable.isMessageDirtyable()`, you should consider all such messages to be dirtyable from the `ContainerRuntime`'s perspective.
+
+If you currently rely on this to suppress `AgentScheduler` ops from impacting last-edited determination, you should move the logic to your `shouldDiscardMessage()` check.
```

Comment:
I've put a comment below about it, but this comment does not reflect that ContainerRuntime.isDirty() calculation has changed, and there is no easy way to augment it when using makeContainerRuntimeWithAgentScheduler